Game/Source/GDKTestGyms/Private/NFRConstants.cpp

```diff
@@ -47,6 +47,7 @@ UNFRConstants::UNFRConstants()
 	, ClientFPSMetricDelay(60)
 	, UXMetricDelay(10 * 60)
 	, ActorCheckDelay(10 * 60)
+	, ActorMigrationCheckDelay(5 * 60)
```

Comment:
Why is the delay not consistent with the UXMetricDelay and the ActorCheckDelay?

Reply:
If we have 10 minutes delay check for migration, and the migration need 5 minutes(9000 ticks) for statistic, I'm worried about it didn't have enough time to report the value before end.